frontend/public/components/utils/status-box.tsx

```diff
@@ -75,6 +75,20 @@ export const EmptyBox: React.FC<EmptyBoxProps> = ({ label }) => {
 };
 EmptyBox.displayName = 'EmptyBox';
 
+export const PVsLoading: React.FC<EmptyBoxProps> = ({ label }) => {
+  const { t } = useTranslation();
+  return (
+    <Box>
+      <div data-test="empty-message" className="text-center">
+        {label
+          ? t('public~PVs are being provisioned on {{label}}, please wait', { label })
+          : t('public~PVs are being provisioned, please wait')}
+      </div>
+    </Box>
+  );
+};
+PVsLoading.displayName = 'PVsLoading';
+
```

Comment:
Needs to be specific to the OCS component. So move it to ceph plugin.

Reply:
Done. Please take a look